lib/oli_web/templates/lti/register.html.eex

```diff
@@ -121,6 +121,7 @@
                     $('.auth_token_url').val(full_url(config, 'auth_token'));
                     $('.auth_login_url').val(full_url(config, 'auth_login'));
                     $('.auth_server').val(full_url(config, 'auth_server'));
+                    $('.line_items_service_domain').val(full_url(config, 'line_items_service_domain'));
```

Comment:
Is it necessary to require this from an admin when registering? Or can we somehow update it from the lti launch params.

My concerns are 2, first we are requiring the admin to correctly provide another key piece of information and there appears to be no way to change/update this by the LMS admin. If we can automate this, I think that would be the best approach